client/my-sites/domain-tip/index.jsx

```diff
@@ -104,7 +111,7 @@ class DomainTip extends React.Component {
 export default connect( ( state, ownProps ) => {
 	const site = getSite( state, ownProps.siteId );
 	const siteSlug = getSiteSlug( state, ownProps.siteId );
-	const queryObject = getQueryObject( site, siteSlug );
+	const queryObject = getQueryObject( site, siteSlug, ownProps.vendor );
```

Comment:
Looks like `ownProps.vendor` is `undefined` at this point - I'm guess the default props are not being applied yet or something at this point? As a result, the `vendor` is not passed at all downstream.

Reply:
Fixed - turns out I needed to provide `defaultProps` to the connected component, not its parent.